packages/@icon-magic/config-reader/src/schemas/config-schema.ts

```diff
@@ -42,8 +42,8 @@ const distributeConfigProperties = {
   svg: {
     type: ['object', null],
     properties: {
-      spriteName: {
-        type: ['string', 'null']
+      spriteNames: {
+        type: ['array', 'null']
       },
       toSprite: {
```

Comment:
Remind me again why we have spriteNames and toSprite as two different properties in the config?

Reply:
I guess so sprite generation can be the default. Currently if we want no sprite generation we can do:

`toSprite: false`

if we only have one prop (and it would have to be `spriteNames` then we would indicate we don't want sprite generation by `spriteNames: []` or `spriteNames: null` which doesn't really communicate that well.